parts/windowsProvision.ps1

```diff
@@ -349,6 +355,36 @@ function Install-PSW {
     }
 }
 
+function Install-AzureDCAP{
+    $installDir = Join-Path $PACKAGES_DIRECTORY "AzureDCAP"
+    Rename-item -Path $PACKAGES["AzureDCAP"]["local_file"] -NewName $PACKAGES["AzureDCAP"]["renamed_file"]
+    Install-ZipTool $PACKAGES["AzureDCAP"]["renamed_file"] `
+                    -InstallDirectory $installDir
+    $p = Start-Process powershell -Wait -NoNewWindow -PassThru -argument "$installDir\script\InstallAzureDCAP.ps1"
+    if($p.ExitCode -ne 0) {
+        Throw "Failed to Add Azure-DCAPLibrary. Please Add it manually."
+    }
+
+}
+
+function Add-RegistrySettings {
+    $AddingRegistryCMD = @"
+    reg query HKLM\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters /v SGX_Launch_Config_Optin
+    if %ERRORLEVEL% EQU 1 goto SETUP

```

Comment:
In the script at the top (AddRegistryForWS2019.cmd), this is a different condition (1 instead of 0). Is that supposed to be the case?

Also, is there a reason this script exists in two places? Do they serve different purposes?

Reply:
errorlevel=0 means the value exists
errorlevel=1 means the value does not exist

The cmd script served no purpose. Yes, I need to get rid of it. I tested the PowerShell script on VMs so the changes in the PowerShell script are correct.

Reply:
Nice, Shruti! Thank you :)